src/sentry/incidents/logic.py

```diff
@@ -652,7 +652,10 @@ def create_alert_rule(
                 )
 
             try:
-                send_historical_data_to_seer(alert_rule=alert_rule, project=projects[0])
+                resp = send_historical_data_to_seer(alert_rule=alert_rule, project=projects[0])
```

Comment:
nit: Can we instead update `send_historical_data_to_seer` to return a boolean?
Or maybe we can just have it return an alert rule status?
if bool, we could rename this variable to be `backfill_success` or something

I just find it odd needing to inspect a `response` status here 🤔